content/es/docs/concepts/services-networking/ingress.md

```diff
@@ -737,7 +737,7 @@ recurso Ingress:
 * Usa
   un [Service.Type=LoadBalancer](es/docs/concepts/services-networking/service/#loadbalancer)
 * Usa
-  un [Service.Type=NodePort](es/docs/concepts/services-networking/service/#nodeport)
+  un [Service.Type=NodePort](es/docs/concepts/services-networking/service/#type-nodeport)
```

Comment:
The correct reference is different in the Spanish `service.md` file, you can see it [here](https://github.com/kubernetes/website/blob/main/content/es/docs/concepts/services-networking/service.md?plain=1#L406).

```suggestion
  un [Service.Type=NodePort](es/docs/concepts/services-networking/service/#tipo-nodeport)
```

Reply:
Done, thanks